wire-ios-sync-engine/Source/SessionManager/SessionManager+VoIPPushManagerDelegate.swift

```diff
@@ -39,21 +39,21 @@ extension SessionManager: VoIPPushManagerDelegate {
             let activity = BackgroundActivityFactory.shared.startBackgroundActivity(
                 withName: "\(payload.stringIdentifier)",
                 expirationHandler: { [weak self] in
-                  Logging.push.warn("Processing push payload expired: \(payload)")
+                  WireLogger.notifications.warn("Processing push payload expired: \(payload)")
                   self?.notificationsTracker?.registerProcessingExpired()
                 }
             )
         else {
-            Logging.push.warn("Aborted processing of payload: \(payload)")
+            WireLogger.notifications.warn("Aborted processing of payload: \(payload)")
             notificationsTracker?.registerProcessingAborted()
             return completion()
         }
 
         withSession(for: account, perform: { userSession in
-            Logging.push.safePublic("Forwarding push payload to user session with account \(account.userIdentifier)")
+            WireLogger.notifications.info("Forwarding push payload to user session with account \(account.userIdentifier)")
```

Comment:
we can also add the safePublic attributes here

Reply:
true, but I skipped it for now because I could finally merge. will do so with the next change